composer/algorithms/mixup/mixup.py

```diff
@@ -54,8 +52,9 @@ def mixup_batch(input: torch.Tensor,
 
     Returns:
         input_mixed (torch.Tensor): batch of inputs after mixup has been applied
-        target_mixed (torch.Tensor): labels after mixup has been applied
-        perm (torch.Tensor): the permutation used
+        target_orig (torch.Tensor): The original labels
```

Comment:
Fairly strong preference for not returning `target_orig`, since the user has to already have it right there as a variable in order to call this function. Just adds code + makes the return value even longer.

Reply:
Curious to heard @A-Jacobson's thoughts on this. I don't have a strong opinion either way, so unless he does, I'll change it.

Reply:
Ok, changed it so `mixup_batch` no longer returns the original targets.